recipe/meta.yaml

```diff
@@ -1,40 +1,48 @@
 {% set name = "jupyterlab_pygments" %}
-{% set version = "0.1.2" %}
+{% set version = "0.2.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: cfcda0873626150932f438eccf0f8bf22bfa92345b814890ab360d666b254146
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jupyterlab_pygments-{{ version }}.tar.gz
+  sha256: 7405d7fde60819d905a9fa8ce89e4cd830e318cdad22a0030f7a901da705585d
 
 build:
   number: 0
-  noarch: python
-  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
+  skip: True # [py<37]
 
 requirements:
   host:
     - python
     - pip
+    - jupyter-packaging >=0.10,<2
```

Comment:
`~=0.10` means `>=0.10,<1` afaiu, and the `<2` upstream is redundant

Reply:
corrected, thanks